esp/src/src-react/components/InfoGrid.tsx

```diff
@@ -149,18 +151,32 @@ export const InfoGrid: React.FunctionComponent<InfoGridProps> = ({
         setFilterCounts(filterCounts);
     }, [errorChecked, exceptions, infoChecked, otherChecked, warningChecked]);
 
-    return <HolyGrail
-        header={<CommandBar items={buttons} farItems={copyButtons} />}
-        main={
-            <FluentGrid
-                data={data}
-                primaryID={"id"}
-                alphaNumColumns={{ Name: true, Value: true }}
-                columns={columns}
-                setSelection={setSelection}
-                setTotal={setTotal}
-                refresh={refreshTable}
-            ></FluentGrid>
+    React.useEffect(() => {
+        if (mounted) {
+            const header = document.querySelector(".ms-DetailsList-headerWrapper");
+            const viewport = document.querySelector(".ms-Viewport");
+            if (header && viewport) {
+                header.remove();
+                header["style"].top = "-4px";
+                viewport.prepend(header);
+            }
         }
-    />;
+    });
```

Comment:
Without the array of values to "react to" it will get called every iteration...)